build_test.go

```diff
@@ -117,17 +127,21 @@ func testBuild(t *testing.T, context spec.G, it spec.S) {
 			})
 			Expect(err).NotTo(HaveOccurred())
 
+			// I don't think there's any way to guess this or apply a matcher instead of checking equality
+			layer.Path = result.Layers[0].Path
```

Comment:
Not sure if I understand what this line is for. Wouldn't you want something like
`Expect(result.Layers[0].Path).To(Equal(layer.Path))`?

Reply:
I would prefer that, but it looks like the `layer.Path` is autogenerated and I don't think there's a way to check equality. I've (slightly) modified how this is done and the comment that goes with it, could you take another look?